app/views/shared/_user-details.html.erb

```diff
@@ -0,0 +1,14 @@
+<h2><%= current_user.first_name %> <%= current_user.last_name %></h2>
+<p><%= current_user.age_range %>, <i class="fas fa-venus"></i></p>
```

Comment:
The gender icon was a placeholder for the pitch. Either we have to connect it to the user profile or delete it. (I'm ok deleting it.)

Reply:
@worksofgenius That depends on whether we want our user to have a gender attribute or not. It's not too complicated to add. What do you and @attinore think?

Reply:
@zuccamia At this point, I think skills are more important than gender.